nailgun/entities.py

```diff
@@ -4205,27 +4205,61 @@ def download_debug_certificate(self, synchronous=True, **kwargs):
         return _handle_response(response, self._server_config, synchronous)
 
 
-class OSDefaultTemplate(Entity):
+class OSDefaultTemplate(
+        Entity,
+        EntityCreateMixin,
+        EntityDeleteMixin,
+        EntityReadMixin,
+        EntitySearchMixin,
+        EntityUpdateMixin):
```

Comment:
Please provide some test results: commandline output for each added operation - create/read/search/update/delete to ensure every single operation works and no more workarounds (like custom `update_payload` method you've already implemented) are needed.

Reply:
Where do you want the results? What do you mean by commandline? IPython?
Can you give me some pointers?

Reply:
Yes, ipython, `entities.OSDefaultTemplate().create()`, `entities.OSDefaultTemplate(id=id).read()` etc.

Reply:
Just a comment here containing iPython's output should be enough. Pointers - any PR containing new mixins for entities, for example:
https://github.com/SatelliteQE/nailgun/pull/399#issue-222688398
https://github.com/SatelliteQE/nailgun/pull/446#issue-259765994

Reply:
That explains, why i did not find any in the source...
Added at the bottom. 👇